source/conf.py

```diff
@@ -20,6 +20,7 @@
 project = 'FIRST Robotics Competition'
 copyright = '2019, FIRST'
 author = 'WPILib'
+version = '2019.4.1'
```

Comment:
Do we actually want these docs to track WPILib releases? I'd imagine the only thing that'd change between season updates is any release notes.

Reply:
Probably not. It feels like one more thing we have to remember and coordinate while working on releases for not much win.

Reply:
EPUB generation requires a version be set. An alternative could be years. 2019, 2020, etc. Thoughts?

Reply:
Hmmm...yeah, just year feels more representative than WPILib version which I don't think we will keep up to date. But we could also just fill in a garbage version (like 1.0 or something).